tox.ini

```diff
@@ -64,6 +64,18 @@ commands =
     pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} \
         --durations=0 --asyncio-mode=auto
 
+
+[testenv:shell]
+description = opens a python shell with all the necessary integration test dependencies
+deps =
+    pytest
+    juju
+    psycopg2-binary
+    pytest-operator>0.17.0
+    -r{toxinidir}/requirements.txt
+commands =
+    python3
```

Comment:
Nothing wrong with this, but I would be wary to add a 3rd way of having a shell. Suggest using `poetry` or just leave the `venv` with `requirements-dev.txt`

Reply:
agreed, removed